src/tabbar.cpp

```diff
@@ -127,6 +129,32 @@ void TabBar::setNewTabButton()
     setTabButton(idx, QTabBar::RightSide, Q_NULLPTR);
 }
 
+int TabBar::realTabCount()
+{
+    // The last tab is "+" in TabBar, but that isn't a real tab which displays any content hence the real count is tab count - 1
```

Comment:
Better move (or repeat) this comment at the declaration of `realTabCount()`

Reply:
Repeated.